embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/spans/EmbraceSpanImpl.kt

```diff
@@ -42,8 +42,8 @@ public class EmbraceSpanImpl(
     private var updatedName: String? = null
     private val systemEvents = ConcurrentLinkedQueue<EmbraceSpanEvent>()
     private val customEvents = ConcurrentLinkedQueue<EmbraceSpanEvent>()
-    private val systemAttributes = ConcurrentHashMap<AttributeKey<String>, String>().apply {
-        putAll(spanBuilder.getFixedAttributes().associate { it.key.attributeKey to it.value })
+    private val systemAttributes = ConcurrentHashMap<String, String>().apply {
+        putAll(spanBuilder.getFixedAttributes().associate { it.key.attributeKey.key to it.value })
```

Comment:
Some system attributes now have arbitrary string keys (i.e. session properties), so we make the key a string and reference the string inside the `AttributeKey` instead of requiring that